en/components/select.md

```diff
@@ -5,25 +5,18 @@ _keywords:  angular select, igniteui for angular, infragistics
 ---
 
 ## Select
-The Ignite UI for Angular Select component allows a single item selection from a dropdown list of items with specialized positioning. This Material based form control offers quick items list navigation, including selection based on a single or multiple characters match. 
+<p class="highlight">The [IgxSelectComponent]({environment:angularApiUrl}/classes/igxselectcomponent.html) allows a single selection from a list of items, placed in a dropdown. This Material-based form control offers quick items list navigation, including selection, based on a single or multiple characters match. Since the `igx-select` extends the`igx-drop-down`, you can handle all of the dropdown events.</p>
```

Comment:
```suggestion
<p class="highlight">The [Select Component]({environment:angularApiUrl}/classes/igxselectcomponent.html) allows a single selection from a list of items, placed in a dropdown. This Material-based form control offers quick items list navigation, including selection, based on a single or multiple characters match. Since `IgxSelectComponent` extends `IgxDropDownComponent`, you can handle all of the Drop Down events in the Select instance as well.</p>
```